src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -8,34 +8,60 @@
 import com.revrobotics.CANSparkMax;
 
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.classes.DebugLog;
+import frc.robot.classes.TimeOfFlightSensor;
 import frc.robot.hardware.MotorController;
 import frc.robot.hardware.MotorController.MotorConfig;
+import frc.robot.subsystems.ArmSubsystem.ArmState;
 import edu.wpi.first.networktables.GenericEntry;
 import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
 import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardTab;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 
 public class IntakeSubsystem extends SubsystemBase implements AutoCloseable {
+  public enum FlightStates{
+    IDLE,
+    CUBE,
+    CONE,
+    CONE_SCORE
+  }
+
   public static final double kConeIntakeSpeed = -0.75;
   public static final double kConeOuttakeSpeed = 0.75;
   public static final double kCubeIntakeSpeed = 0.55;
   public static final double kCubeOuttakeSpeed = -0.55;
+  private TimeOfFlightSensor timeOfFlightSensor;
+
+  private final double coneActivationThreshold = 550.0; // placeholder value for how small values have to be for cone to be there
```

Comment:
The values are actually integers returned from the sensor, would be best to keep this an integer as well

Reply:
Adding a "TODO" string will help the compiler flag this as a placeholder.  But I think we can remove that comment and use 550